src/ExpressionEvaluator/Core/Source/ExpressionCompiler/PDB/MethodDebugInfo.Native.cs

```diff
@@ -135,18 +130,38 @@ public override int GetHashCode()
                         out importRecordGroups,
                         out externAliasRecords);
 
-                    ReadCSharpNativeCustomDebugInfo(
-                        symReader,
-                        methodToken,
-                        methodVersion,
-                        allScopes,
-                        out hoistedLocalScopeRecords,
-                        out dynamicLocalMap,
-                        out dynamicLocalConstantMap,
+                    defaultNamespaceName = "";
+                }
+
+                hoistedLocalScopeRecords = ImmutableArray<HoistedLocalScopeRecord>.Empty;
```

Comment:
Could those `null` initializers could be inlined to the definition of those locals?